src/main/resources/application.properties

```diff
@@ -1,7 +1,10 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/books
-spring.datasource.username=postgres
-spring.datasource.password=123456
-spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+#spring.datasource.url=jdbc:postgresql://localhost:5432/books
+spring.datasource.url = jdbc:h2:mem:books
```

Comment:
Test should not modify how we run our code. If you keep this code commented, the application will never start. Why do you need to do this?